src/lib/y2partitioner/actions/edit_blk_device.rb

```diff
@@ -243,25 +248,27 @@ def generic_unusable_error
           name: device.name)
       end
 
-      # Whether the device is an extended partition
+      # Warning message if trying to edit an LVM Snapshot, which is not wrong but weird.
       #
-      # @return [Boolean]
-      def extended_partition?
-        device.is?(:partition) && device.type.is?(:extended)
-      end
+      # @return [Boolean] true if device is not an LVM snapshot or the user confirms to continue
+      def confirm_editing_lvm_snapahot
+        return true unless device.is?(:lvm_snapshot)
+
+        # TRANSLATORS: Error message when trying to edit an LVM LV snapshot. %{name} is
+        # replaced by a logical volume name (e.g., /dev/system/user-data-snapshot)
+        message = format(
+          _("The device %{name} is a LV snapshot volume.\n Do you really want to edit it?"),
```

Comment:
I would say "an LVM snapshot volume" or simply "a snapshot volume". LVM is a common acronym that is understood as a proper name, but LV is not.

Reply:
And maybe we should indicate the origin. Something like `%{name} is a snapshot of %{origin}".